### DISCUSSION TITLE
Unable to install custom TLS certificate on Device Image v13.1 / Android 10.0

### DISCUSSION THREAD

**Discussion by zvymazal** [Genymobile/feedback#56] [Genymotion Device Image] [answered]
Hello everyone, I'd like to raise this issue that we have encountered and which so far were unable to solve. I tried to install custom TLS certificate on the newest AWS AMIs (v13.1) for Android 10.0 according to the instruction in this repository: https://github.com/Genymobile/genymotion-cloud-ssl-tool/. However, I have been unsuccessful so far. In the web interface I get the following information about the certificate:

```
Certificate information
  Distinguish Name:
  Alternative Names:
  Type: (X.509)
  Expiration: Invalid Date ("P0D")
  Serial Number:
```

We've been using AWS AMIs for some years now and this has never been a problem. Same procedure works fine for older device images (currently on v12).

Any ideas what might be wrong?

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @zvymazal,
We are sorry but this discussion board is for improvement suggestions and feature requests only ; it is not suitable for technical assistance requests.
Please file a ticket to our technical support team, either using the Help widget from the instance, or directly from our help center at https://support.genymotion.com/hc/en-us/requests/new
Thank you for your understanding.

**Reply by zvymazal** [reply]
Thank you @jmaneyrol69 🙇 I shall do that!
